src/pages/TransferConfirm/TransferConfirm.jsx

```diff
@@ -0,0 +1,122 @@
+import "./TransferConfirm.scss";
+import Button from "../../components/Button/Button";
+import arrow from "../../assets/images/transfer-focus-inverted.png";
+
+const TransferConfirm = ({
+  sendAmount = 1000.0,
+  sender = "Samantha Brooks",
+  senderAcc = "12345678",
+  senderSort = "123456",
+  totalToSend = "£100.00",
+  receiver = "Erik Just",
+  receiverAcc = "87654321",
+  receiverSort = "654321",
+  fundsRem = "£1",
+  rate = 1.3595,
+  fee = 20.0,
```

Comment:
Please remove the values of these, I get they are used for testing but this is going to be dynamic so don't set any of them to default values